code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1066,6 +1062,18 @@
 
 	return .
 
+/obj/item/bodypart/proc/huskify_image(image/thing_to_husk, draw_blood = TRUE)
+	var/husk_color_mod = rgb(96, 88, 80)
+	var/icon/husk_icon = new(thing_to_husk.icon)
+	husk_icon.ColorTone(husk_color_mod)
+	thing_to_husk.icon = husk_icon
+	if(draw_blood)
+		var/mutable_appearance/husk_blood = mutable_appearance(icon_husk, "[husk_type]_husk_[body_zone]")
+		husk_blood.blend_mode |= BLEND_INSET_OVERLAY
+		husk_blood.appearance_flags |= RESET_COLOR
+		husk_blood.dir = thing_to_husk.dir
+		thing_to_husk.add_overlay(husk_blood)
```

Comment:
I think this is okay? it's baked directly onto the limb code but we shouldn't get more like these

Reply:
Does curing clear the husks overlay? Or do they stack infinitely? You're technically adding overlays outside of the usual path

Reply:
clears the overlay